scripts/airgap.sh

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+# ==================================================================================================
+# TODO:
+#
+# Review (Victor): https://github.com/kubewarden/kwctl/pull/375#discussion_r1048740479
+# current approach doesn't use the GH release files kubewarden-crds-1.2.3_kubewarden-defaults-1.2.8_kubewarden-controller-1.2.8_policylist.txt
+# - it does
+# canonical locations of artifacts
+# - please explain (helm charts vs release?)
+#
+# TODO:
+# fix https://docs.kubewarden.io/operator-manual/airgap/requirements
+# blogpost - https://github.com/kubewarden/kubewarden.io/pull/153
+# fix documentation - https://github.com/kubewarden/docs/pull/150
+# - get image / policy list from pulled charts?
+# ==================================================================================================
+
+
+# Defaults
+IMAGES_TAR="kubewarden-images.tar"
+POLICIES_TAR="kubewarden-policies.tar.gz"
+
+PROG=$(basename "$0")
+USAGE="\
+Usage:  $PROG command [--parameters]
+
+  $PROG list [--cert-manager]
+  $PROG pull -l file.json [--dry]
+  $PROG push -l file.json -r registry[:port] [--insecure] [--dry]
+  $PROG install -l file.json -r registry[:port] [--insecure] [--dry]
+
+Parameters:
+  -l|--list       Json file with list of required dependencies
+  -r|--registry   Private registry where artifacts are pushed. Helm charts are configured for this registry
+  --dry           Do not perform actions, only print commands that would be executed
+  --cert-manager  Include cert-manager dependencies in the generated list
+  --insecure      For pull command skip kwctl certificate verification (an alternative to sources.yaml file)
+                  For install disable policyServer verification of registry
+
+Commands:
+  list     Generate json list of dependencies. You can redirect it to a file
+  pull     Pull components from list to current directory
+  push     Push components in list from current directory to local registry
+  install  Install helm charts configured with local registry
```

Comment:
While I like the idea of providing a command for installing the helm charts, I think it adds a new layer of indirection for users, as they aren't aware of what exactly has happened there, and they need to read outputs.
Also, I would expect people to use GitOps to deploy their Helm charts to the cluster. They would just commit `values.yaml`, `insecure-sources.yaml` and similar into git, and then call an install job.

Maybe instead of having an `install` subcommand that calls `helm install`, we can provide a `scaffold-install` subcommand, that creates a values.yaml and insecure-sources.yaml or so as needed. And then the user can review them, change them if needed, and make the call to `helm install` or `upgrade -i` themselves with whatever tooling they have. This gives them the correct values.yaml pointing the internal registry, disabling the recommendedPolicies, and so, and they can review it and be aware of what are they are going to get. We could even document the scaffolded values.yaml with the reasons; for example "disabling recommendedPolicies, you can re-enable them if you have mirrored the wasm modules in your air-gapped registry".